extension/src/webpages/registerStudent.js

```diff
@@ -0,0 +1,12 @@
+window.onload = function() {
+    let studentNameField = document.getElementById("studentNameField");
+    studentNameField.onkeydown = function(event) {
+        if (event.code == "Enter" || event.key == "Enter") {
+            $("#studentCollapseCard").collapse("hide");
+            $("#loginConfirmationCollapseCard").collapse("show");
+            chrome.storage.local.set({"studentName": studentNameField.value}, function() {
+                chrome.storage.local.get(console.log);
```

Comment:
let's just remove this.
If you really want a debug statement here, let's try with `console.debug` with additional messages like `student name has been set to ${e}`